README.md

```diff
@@ -15,21 +15,26 @@ client nukleus -> BEGIN with extension data -> nukleus-kafka -> (other nuklei) -
 
 The extension data contains the following fields, all optional:
 
-- `topic name`: if specified, only subscriptions to this topic will be allowed (i.e. BEGIN extension data must specify this topic)
+- `topicName`: if specified, only subscriptions to this topic will be allowed (i.e. BEGIN extension data must specify this topic)
+- `deltaProperty`: if specified, the given message property will be used to generate delta messages (see Delta Messages below)
```

Comment:
The Kafka terminology is `header`, not `property`, right?

Reply:
Good catch, corrected in this PR and the spec one